DB Setup Seeder/app.js

```diff
@@ -3,42 +3,49 @@ const path = require("path");
 const express = require("express");
 const bodyParser = require("body-parser");
 const sequelize = require("./util/database");
-const User = require("./models/user");
 const Associations = require("./models/associations")();
 
+const userRoutes = require("./routes/user");
```

Comment:
don't have these many routes here. have it in a common router and import these there.

Reply:
ok satheesh, will make a common router file